.circleci/config.yml

```diff
@@ -216,6 +229,13 @@ jobs:
               package_cloud push choria/${REPO}/el/${d} ${i}
             done
           done
+          for d in buster
+          do
+            for i in $(find /tmp/workspace -name \*.${d}.\*.deb)
```

Comment:
I could not find what I an supposed to be searching here, so maybe this need adjustment…

Reply:
yeah it would be the same as rake build, I tend to limit what I stick on nightly because the repo grows quite a lot quite quickly so maybe try to limit the find to the arm ones for now

Reply:
Oh, I see, I didn't realise there was no Debian nightly releases, and the problem does not exist with CentOS / RedHat.